task-13/src/main/java/com/epam/winter_java_lab/task_13/domain/User.java

```diff
@@ -0,0 +1,132 @@
+package com.epam.winter_java_lab.task_13.domain;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.validator.constraints.Length;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import javax.persistence.*;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.Collection;
+import java.util.Set;
+
+@Entity
+@Table(name = "usr")
+@EqualsAndHashCode(of = {"id"})
+public class User implements UserDetails {
+    @Getter
+    @Setter
```

Comment:
Could you use @Data annotation from lombok instead of multiple getter and setter annotations?